src/main/kotlin/io/github/kryszak/gwatlin/api/gamemechanics/model/skill/SkillSlot.kt

```diff
@@ -19,5 +19,7 @@ enum class SkillSlot {
     @SerialName("Utility")
     UTILITY,
     @SerialName("Weapon_[1-5]")
-    WEAPON1_5
+    WEAPON1_5,
+    @SerialName("Elite")
+    ELITE,
```

Comment:
Please add test case to be 100% sure it'll work

Reply:
I added a new test file `SkillsClientTest` and two sample json responses. I followed the structure of `ItemsClientTest`, but I didn't have time to add many assertions.

Reply:
There we go, some assertions. I noticed also that e.g. Revenant's "Selfish Spirit" has a slot `HEAL`. I added that as well. 

I noticed that Traited Fact type is not quite complete, but I'll create a separate issue for that.